code/modules/mob/living/simple_animal/abnormality/waw/sphinx.dm

```diff
@@ -0,0 +1,469 @@
+	inspuz: quest	<br><br>
+	-K-	<br>
+	kas: no	<br>
+	kom: show	<br><br>
+	-L-	<br>
+	lemanto: human	<br>
+	leti: gave	<br>
+	luvri : open	<br><br>
+	-M-	<br>
+	manba: came	<br>
+	medon: kill	<br>
+	mina: water	<br>
+	mos: coin	<br>
+	mi: I	<br><br>
+	-N-	<br>
+	naar: fire	<br>
+	neu: papyrus/textile	<br><br>
+	-P-	<br>
+	por: beggar	<br>
+	prin: consume	<br>
+	puwun: to	<br><br>
+	-Q-	<br>
+	quir: translate	<br>
+	quistra: body/armor	<br><br>
+	-S-	<br>
+	sagmo: begone	<br>
+	sinco: honor/value	<br>
+	sommel: weapon/stick	<br><br>
+	-T-	<br>
+	tai: the	<br>
+	thran: angry	<br>
+	tolly: stupid	<br>
+	tom: to 	<br><br>
+	-U-	<br>
+	ucaf: lands	<br>
+	utast: abnormality/shell"}
+
+//chems
+/datum/reagent/medicine/ichor //from jar
+	name = "Ichor"
+	description = "an anomalous substance"
+	reagent_state = LIQUID
+	color = "#D2FFFA"
+	metabolization_rate = 0.5 * REAGENTS_METABOLISM
+	overdose_threshold = 30
+
+/datum/reagent/medicine/ichor/on_mob_metabolize(mob/living/L)
+	..()
+	L.add_movespeed_modifier(/datum/movespeed_modifier/reagent/ichor_boost)
+
+/datum/reagent/medicine/ichor/on_mob_end_metabolize(mob/living/L)
+	L.remove_movespeed_modifier(/datum/movespeed_modifier/reagent/ichor_boost)
+	..()
+
+/datum/reagent/medicine/ichor/overdose_process(mob/living/carbon/M)
+	if(prob(25))
+		M.reagents.remove_reagent(type, metabolization_rate*15)
+		M.vomit(20, TRUE, TRUE, 1, FALSE, FALSE, FALSE, TRUE) //copied from vomitting extra organs
+
+/datum/movespeed_modifier/reagent/ichor_boost
+	multiplicative_slowdown = -0.25
+
+/datum/reagent/medicine/theonic_gold //from needle
+	name = "Theonic gold"
+	description = "an anomalous substance"
+	reagent_state = LIQUID
+	color = "#D2FFFA"
+	metabolization_rate = 0.5 * REAGENTS_METABOLISM
+	overdose_threshold = 30
+
+/datum/reagent/medicine/theonic_gold/on_mob_metabolize(mob/living/L)
+	..()
+	ADD_TRAIT(L, TRAIT_STUNRESISTANCE, type)
+	L.adjustStaminaLoss(-10, 0)
+
+/datum/reagent/medicine/theonic_gold/on_mob_end_metabolize(mob/living/L)
+	REMOVE_TRAIT(L, TRAIT_STUNRESISTANCE, type)
+	..()
+
+/datum/reagent/medicine/theonic_gold/overdose_process(mob/living/carbon/M)
+	if(prob(3) && iscarbon(M))
+		M.visible_message("<span class='danger'>[M] starts having a seizure!</span>", "<span class='userdanger'>You have a seizure!</span>")
+		M.Unconscious(100)
+		M.Jitter(350)
+
+//objects - effects
+/obj/effect/temp_visual/eyewarning
+	name = "warning"
+	desc = "LOOK AWAY!."
+	icon = 'ModularTegustation/Teguicons/tegu_effects.dmi'
+	icon_state = "sphinx_eye"
+	move_force = INFINITY
+	pull_force = INFINITY
+	generic_canpass = FALSE
+	movement_type = PHASING | FLYING
+	layer = POINT_LAYER	//High enough that you can see it if you look up.
+	light_color = COLOR_YELLOW
+	light_range = 5
+	light_power = 7
+	duration = 1.2 SECONDS
+
+/obj/effect/temp_visual/eyewarning/Initialize()
+	..()
+	animate(src, pixel_x = 0, pixel_z = 32, time = 0 SECONDS)
+
+/mob/living/simple_animal/hostile/abnormality/sphinx/PickTarget(list/Targets)
```

Comment:
THIS needs to be relocated, it's in the middle of a bunch of objects and very far away from Sphinx's working code.